config/rbac/oauth_proxy_role.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: oauth-proxy-role
+rules:
+- apiGroups: ["authentication.k8s.io"]
+  resources:
+  - tokenreviews
+  verbs: ["create"]
```

Comment:
pls follow the shape of other rbac files, like
```
verbs:
 - create
```
same for the rest

Reply:
if i use create here it's not working as expected i have tried and tested this.